bluesky_queueserver/manager/start_manager.py

```diff
@@ -188,24 +188,32 @@ def run(self):
 
 def start_manager():
     parser = argparse.ArgumentParser(
-        description="Start a RE Manager", epilog=f"blueksy-queueserver version {__version__}"
+        description="Start Run Engine (RE) Manager", epilog=f"blueksy-queueserver version {__version__}"
     )
     parser.add_argument(
-        "--zmq_addr",
+        "--zmq-addr",
         dest="zmq_addr",
         type=str,
         default="tcp://*:5555",
         help="The address of ZMQ server.",
     )
-    parser.add_argument(
-        "--profile_collection",
-        "-p",
-        dest="profile_collection_path",
+
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument(
+        "--startup-dir",
+        dest="startup_dir",
+        type=str,
+        help="Path to directory that contains startup files (profile collection).",
```

Comment:
I think `(profile collection)` might be confusing here because it might be not be obvious whether `~/.ipython/profile_collection` or `~/.ipython/profile_collection/startup` are expected. Is it too verbose to include `(example: ~/.ipython/profile_collection/startup)`

Reply:
`--startup-dir` may be any accessible directory, which contains meaningful `*.py` and `*.ipy` files. I changed the help string to reflect this point.